lib/config-generator.js

```diff
@@ -181,6 +181,7 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
         }
 
         if (this.answers.framework === "react") {
+            exportContent += "  { files: [\"**/*.jsx\"], languageOptions: { parserOptions: { ecmaFeatures: { jsx: true } } } },\n";
```

Comment:
it should have been configured in the react plugin, but as the new config defaults, it was only applied to `*.{js,mjs,cjs}`. Or we can change it to `*.{js,mjs,cjs,jsx}`?

Reply:
> Or we can change it to `*.{js,mjs,cjs,jsx}`?

Makes sense to me 👍